webapp/src/app/components/leaderboard/leaderboard.component.html

```diff
@@ -0,0 +1,55 @@
+<div class="flex flex-col gap-4 py-8">
+  <h1 class="text-3xl font-bold">Artemis Leaderboard</h1>
+  @if (query.isPending()) {
+    <span class="text-muted-foreground">Data is loading...</span>
+  } @else if (query.error()) {
+    <span class="text-destructive">An error has occurred</span>
+  }
+
+  @if (leaderboard(); as leaderboard) {
+    <div class="px-4">
+      <app-table>
+        <thead appTableHeader>
+          <tr appTableRow>
+            <th appTableHead>Contributor</th>
+            <th appTableHead>Score</th>
+            <th appTableHead>Activity</th>
+          </tr>
+        </thead>
+        <tbody appTableBody>
+          @for (entry of leaderboard; track entry.githubName) {
+            <tr appTableRow>
+              <td appTableCell>
+                <a href="https://github.com/{{ entry.githubName }}" target="_blank" rel="noopener noreferrer" class="flex items-center gap-2 font-medium">
+                  <img src="{{ entry.avatarUrl }}" width="24" height="24" />
+                  <span class="text-muted-foreground">{{ entry.name }}</span>
+                </a>
+              </td>
+              <td appTableCell>{{ entry.score }}</td>
+              <td appTableCell class="flex items-center gap-4" title="Changes Requested">
+                <div class="flex items-center gap-2">
+                  <app-icon-pull-request-changes-requested />
+                  {{ entry.changesRequested }}
+                </div>
```

Comment:
Have you checked how it looks when you remove the `0` value activities with `@if`? I think it would look good

Reply:
This is how it would look for the past 7 days:
![image](https://github.com/user-attachments/assets/d6948071-e1e5-4755-9c0a-b54be4892dfa)
It will be much more filled when we extend the timeframe to 30days of course, but I also like it more.

Reply:
Looks really good :) 
On that note, why is Kaan having score 68 without having any activity? Are we missing something?